src/main/java/org/opentripplanner/framework/token/TokenFormat.java

```diff
@@ -0,0 +1,13 @@
+package org.opentripplanner.framework.token;
+
+import org.opentripplanner.framework.text.CharacterEscapeFormatter;
+
+interface TokenFormat {
+  char FIELD_SEPARATOR = '|';
+  char TOKEN_ESCAPE = '\\';
+  char TOKEN_SUBSTITUTION = '+';
+
+  static CharacterEscapeFormatter tokenFormatter() {
```

Comment:
The `CharacterEscapeFormatter` is a "generic" implementation in the framework package, and this interface only work as a configurator of that - Creating an instance for the Token serialization. I am not convinced this is a good idea - I would prefare to use a library for the escaping - we are using CSV libs elsewhere, so I will have a look and se if I can use it here. The only thing is that I do not want the `"` to be escaped - only the field separator( `|`). I used pipe, but we can use any thing like `;`. Most IDs have `:`, so I choose pipe since it give the best contrast, not (`,`, `;`, `\t`).  

To make my intention a bit more clear I can make the `TokenFormat` a utility class instead, all it does is to create a `CharacterEscapeFormatter` (generic implementation in the framework). What do you think?

Reply:
I did not find something I can use, so I will stick with this. I will change the interface to a utility class, maybe TokenFormatConfigutator is better.

Reply:
That is a clearer solution I think.